torch/_dynamo/variables/torch.py

```diff
@@ -545,6 +545,21 @@ def get_state_from_generator():
 
                     fn_ = sym_sqrt
 
+            if fn_ is torch.tensor:
+                def check_any_unspec(x):
+                    # NB: This includes UnspecializedPythonVariable
+                    if isinstance(x, TensorVariable):
+                        return True
+                    elif isinstance(x, ListVariable):
+                        return any(check_any_unspec(y) for y in x.items)
+                    # TODO: there maybe other recursive structures you need to
+                    # check
+                    else:
+                        return False
+
+                if check_any_unspec(args[0]):
+                    unimplemented("unspec call to torch.tensor")
```

Comment:
Right now, you can't pass SymInt to torch.tensor constructor, because make_fx tracing doesn't know how to deal with it, since we just inline tensor constant into graph. Actually, Dynamo export can deal with it just fine. This is the `int_tensor` thing.

This isn't the real reason I added this code though: originally when we treated unspec ints as 0d tensors, I would sometimes get a torch.tensor([fake_tensor]) call, where the fake tensor represented an unspecialized 0d scalar. This doesn't work, cuz we try to item() the fake tensor to get the real value out but there's no real value. However, you shouldn't really hit this case anymore (unless you make a numpy scalar and pass it to torch.tensor but I REFUSE TO ADD THIS TEST because no one actually is going to hit this in practice.)